doc/admin-guide/files/records.config.en.rst

```diff
@@ -432,6 +432,14 @@ Thread Variables
    HTTP/2 graceful shutdown for now. Stopping Traffic Server here means sending
    `traffic_server` a signal either by `bin/trafficserver stop` or `kill`.
 
+.. ts:cv:: CONFIG proxy.config.thread.max_heartbeat_mseconds INT 60
```

Comment:
Add `:unit: milliseconds` as per other examples. Pays to be very explicit.

Reply:
I added this but github can't seem to find the change.